plugins/workspace-minimap/test/index.ts

```diff
@@ -12,22 +12,23 @@ import * as Blockly from 'blockly';
 import {toolboxCategories, createPlayground} from '@blockly/dev-tools';
 import {Minimap, PositionedMinimap} from '../src/index';
 
-/**
- * Create a workspace.
- * @param blocklyDiv The blockly container div.
- * @param options The Blockly options.
- * @returns The created workspace.
- */
-function createWorkspace(blocklyDiv: HTMLElement,
-    options: Blockly.BlocklyOptions): Blockly.WorkspaceSvg {
-  const workspace = Blockly.inject(blocklyDiv, options);
-
-  const minimap = new Minimap(workspace);
-  minimap.init();
-
-  return workspace;
-}
-
+// Creates the primary workspace and adds the minimap.
 const ws = Blockly.inject('primaryDiv', {toolbox: toolboxCategories});
 const mp = new Minimap(ws);
 mp.init();
+
+// Creates 100 random blocks
+for (let i = 0; i < 100; i++) {
+  const prototype = 'controls_if';
+  const block = ws.newBlock(prototype);
+  block.initSvg();
+  block.render();
+}
+
+// Move the blocks to random locations.
+for (const block of ws.getAllBlocks(true)) {
+  const cord = new Blockly.utils.Coordinate(
+      Math.round(Math.random() * 450 + 40),
+      Math.round(Math.random() * 600 + 40));
+  block.moveTo(cord, ['drag']);
```

Comment:
This isn't being triggered by a drag, I'd probably just omit the optional reason argument here.